r_arm_controller:
  joints:
    - r_shoulder_pan_joint
    - r_shoulder_lift_joint
    - r_upper_arm_roll_joint
    - r_elbow_flex_joint
    - r_forearm_roll_joint
    - r_wrist_flex_joint
    - r_wrist_roll_joint
  test_goals:
    r_shoulder_pan_joint:   {q: 0.3, qdot: 0.0}
    r_shoulder_lift_joint:  {q: 0.0, qdot: 0.0}
    r_upper_arm_roll_joint: {q: 0.0, qdot: 0.0}
    r_elbow_flex_joint:     {q: 0.3, qdot: 0.0}
    r_forearm_roll_joint:   {q: 0.0, qdot: 0.0}
    r_wrist_flex_joint:     {q: 0.3, qdot: 0.0}
    r_wrist_roll_joint:     {q: 0.0, qdot: 0.0}
  test_execution_time: 5.0

l_arm_controller:
  type: "ias_mechanism_controllers/MultiJointVelocityController"
  joints:
    - l_shoulder_pan_joint
    - l_shoulder_lift_joint
    - l_upper_arm_roll_joint
    - l_elbow_flex_joint
    - l_forearm_roll_joint
    - l_wrist_flex_joint
    - l_wrist_roll_joint
  test_goals:
    l_shoulder_pan_joint:   {q: -0.35, qdot: 0.0}
    l_shoulder_lift_joint:  {q: -0.35, qdot: 0.0}
    l_upper_arm_roll_joint: {q: -0.35, qdot: 0.0}
    l_elbow_flex_joint:     {q: 0.0, qdot: 0.0}
    l_forearm_roll_joint:   {q: 0.0, qdot: 0.0}
    l_wrist_flex_joint:     {q: 0.0, qdot: 0.0}
    l_wrist_roll_joint:     {q: 0.0, qdot: 0.0}
  test_execution_time: 5.0

l_arm_vel:
  joint_to_test: "l_shoulder_pan_joint"
  joint_index: 0
  joint_velocity: 0.5 # reasonable max for test with 2s: 1.0
  test_time: 2.0

#  joint_to_test: "l_shoulder_lift_joint"
#  joint_index: 1
#  joint_velocity: 0.3 # reasonable max for test with 2s: 0.6
#  test_time: 2.0

#  joint_to_test: "l_upper_arm_roll_joint"
#  joint_index: 2
#  joint_velocity: 1.0 # reasonable max for test with 2s: 1.8
#  test_time: 2.0

#  joint_to_test: "l_elbow_flex_joint"
#  joint_index: 3
#  joint_velocity: -0.5 # reasonable max for test with 2s: -0.9
#  test_time: 2.0

#  joint_to_test: "l_forearm_roll_joint"
#  joint_index: 4
#  joint_velocity: 1.0 # reasonable max for test with 2s: 2.0
#  test_time: 2.0

#  joint_to_test: "l_wrist_flex_joint"
#  joint_index: 5
#  joint_velocity: -0.5 # reasonable max for test with 2s: -0.9
#  test_time: 2.0

#  joint_to_test: "l_wrist_roll_joint"
#  joint_index: 6
#  joint_velocity: 1.0 # reasonable max for test with 2s: 2.0
#  test_time: 2.0
